packages/enzyme-adapter-react-13/src/ReactThirteenAdapter.js

```diff
@@ -133,6 +133,7 @@ class ReactThirteenAdapter extends EnzymeAdapter {
           };
           const ReactWrapperComponent = createMountWrapper(el, { ...options, adapter });
           const wrappedEl = React.createElement(ReactWrapperComponent, wrapperProps);
+          // eslint-disable-next-line react/no-render-return-value
```

Comment:
let's disable this in each package's .eslintrc (https://github.com/airbnb/enzyme/blob/master/packages/enzyme-adapter-react-13/.eslintrc) rather than inline, since it's disabling being OK depends on the react version.

however, i'm not seeing this as a warning or an error - why include it?